src/Translatable/Translatable.php

```diff
@@ -57,6 +58,10 @@ public function getTranslation($locale = null, $withFallback = null)
         $locale = $locale ?: $this->locale();
         $withFallback = $withFallback === null ? $this->useFallback() : $withFallback;
         $fallbackLocale = $this->getFallbackLocale($locale);
+        
+        if($this->isJoinTranslated()){
```

Comment:
Please put a space between the `if` and the `(`. Also a space before the final `{`

Like this: `if ($this->isJoinTranslated()) {`.